src/layer/vector/Canvas.js

```diff
@@ -215,7 +215,7 @@ L.Canvas = L.Renderer.extend({
 	},
 
 	_onMouseMove: function (e) {
-		if (!this._map || this._map.dragging._draggable._moving || this._map._animatingZoom) { return; }
+		if (!this._map || (this._map.dragging._enabled && this._map.dragging._draggable._moving) || this._map._animatingZoom) { return; }
```

Comment:
Also, `this._map._dragging`, inheriting from `L.Handler` has a `enabled()` method that should be used here instead of private property.